### ISSUE TITLE
UI tweak

### ISSUE THREAD

**Issue by cfb3** [bryce-fujita/PFMS-React#1]
Edit Front end UI to say number of iterations.

**Comment by bryce-fujita** [comment]
Changed wording in both library and ui to refer to iterations.  NOTE: is it worth changing the variable names as well?

**Comment by cfb3** [comment]
Yes, it is worth it to change the variable names. Think of the developers who will come after you!
